fh-messaging/package.json

```diff
@@ -4,10 +4,12 @@
   "version": "3.0.3-BUILD-NUMBER",
   "repository": {
     "type": "git",
-    "url": "dberesford@amz1svn1.feedhenry.com:/var/git/fh-messaging.git"
+    "url": "git://github.com/feedhenry/fh-messaging.git"
   },
-  "author": "Damian Beresford",
-  "license": "Apache-2.0",
+    "bugs": {
```

Comment:
No biggie, right, but I think this line might be indented more than expected, is it?